package/network/services/hostapd/src/src/ap/ubus.c

```diff
@@ -285,6 +292,11 @@ hostapd_bss_del_client(struct ubus_context *ctx, struct ubus_object *obj,
 	if (tb[DEL_CLIENT_DEAUTH])
 		deauth = blobmsg_get_bool(tb[DEL_CLIENT_DEAUTH]);
 
+	if (tb[DEL_CLIENT_ALLOW_CON_COUNT])
+		max_ban_attempts = blobmsg_get_u32(tb[DEL_CLIENT_ALLOW_CON_COUNT]);
```

Comment:
Wouldn't it make more sense to rename `DEL_CLIENT_ALLOW_CON_COUNT` into `DEL_CLIENT_BAN_MAX_ATTEMPTS` (or rename the max_ban_attempts to something else)? It doesn't make sense to name it allow_con_count and assign it to max_ban_attempts variable, I really miss the point here.